frontend/issue.ts

```diff
@@ -216,6 +219,34 @@ export async function getIssueComments(
   });
 }
 
+export async function deleteIssueComments(
+  tx: WriteTransaction,
+  issueID: string
+): Promise<void> {
+  const entries = (await getIssueComments(tx, issueID)) || [];
+  await entries
+    .map((c) => commentKey(c.issueID, c.id))
+    .map((comment) => tx.del(comment));
+}
+
+export async function deleteIssueDescription(
```

Comment:
I am confused. This doesn't seem to delete anything...

Reply:
sorry this is a bozo mistake, i moved a function incorrectly from mutators that I created. should of caught this one when i was reviewing.